richardltyler/vueniverse
# Feature/preview functionality

## *Purpose:*
*What does this merge do? Check all that apply.*


## *Description: What does this PR do?*
*Briefly, but clearly describe what you did, and explain any new code.*
- adds functionality for the `Preview` components to switch the POTD to the previous/next day's photo


## *Instructions for reviewer:*
*Where should the reviewer start?*
- `/views/Preview.vue`

*How should this be manually tested?*
- open the page and click the preview components in the bottom 
- they should scroll to the next or previous day respectively 
- there should be no warnings in the console

*Any background context you want to provide?*
*What are the relevant tickets?*
- closes #17 
- closes #16 

## *Issues:*
*Are there any concerns, issues, or bugs in this branch? If so describe them.*
- still needs to be tested 


## Checklist:
